code/script.js

```diff
@@ -1,47 +1,317 @@
+          <div class="bubble user-bubble">
+            <p>${message}</p>
+          </div>
+          <img src="https://images.pexels.com/photos/4113971/pexels-photo-4113971.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="User" />  
+        </section>
+      `
+    } else if (sender === 'bot') {
+      chat.innerHTML += `
+        <section class="bot-msg">
+          <img src="https://images.unsplash.com/photo-1634909924531-4daae117dbc1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2062&q=80" alt="Bot" />
+          <div class="bubble bot-bubble">
+           <p class="bot-text">${message}</p>
+          </div>
+        </section>
+      `
+    }
+    // This little thing makes the chat scroll to th/e last 
+    //message when there are too many to be shown in the chat box
+    chat.scrollTop = chat.scrollHeight
+  }
+  
+
+  
+const botReply = (msg) => {
+    showMessage(msg,"bot");
+}
+
+const userReply = (msg) => {
+    showMessage(msg,"user");
+}
+
+
+//////////////////
+
+let questionNumber = 1;
+
+//If user choose ambulance case - the answer will return to the message here
+const ambulanceAnswer = (message) => {
+   if (questionNumber === 1) {
+       userReply(message);
+       setTimeout(() => ambulanceCase(message), 500);
+   } else if (questionNumber === 2) {
+       userReply(message);
+       setTimeout(() => confirmAmbulance(message),500 ) 
+       restart();
+   } 
+}
+
+//If user choose minor injury - the answer will return to the message here
+const injuryAnswer = (message) => {
+    if (questionNumber === 1) {
+        userReply(message);
+        setTimeout(() => bookingDoctor(message), 1000); // quesion 2
+    } else if (questionNumber === 2) {
+        userReply(message)
+        setTimeout(() => bookingTime(message), 1000);
+    } else if (questionNumber === 3) {
+        userReply(message);
+        setTimeout(() => doctorOption(message), 1000);
+    } else if (questionNumber === 4) {
+        setTimeout(() => userReply(message), 300);
+        setTimeout(() => confirmDoctor(message), 1000);
+        setTimeout(restart, 1500);
+    }
+}
+
+
+
+///////////////////////////// QUESTION 1 FOR BOTH CASES - START HERE
+//Question 1 - Default Greeting - Show 2 different cases
+const greeting = (message) => {
+    questionNumber = 1;
+    botReply(`Hi, I am Dr.Strange, what can I help with today?`);
+    showInjuryCase();
+  } 
+
+//This function is to be added in question 1 - Show 2 different cases
+function showInjuryCase(message) {
+
+    //Show two button
+    inputWrapper.innerHTML = `
+    <button type="button" id="ambulance"> 🚑 In need of an ambulance</button>
+    <button type="button" id="minor-injury"> 🤕 Minor Injury </button>
+  `; 
+  
+    //Add event listener for "in need of ambulance" button
+     const ambulanceBtn = document.getElementById("ambulance");
+     ambulanceBtn.addEventListener("click", () => {
+      ambulanceAnswer("In need of an ambulance");
```

Comment:
Starting from here, it would be good to look through the indentations in the functions and modify them to fit the Guidelines for how to write good code.